ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh

```diff
@@ -67,7 +70,28 @@ then
   . network.sh
   . utils.sh
 
-  ssh -o 'ConnectTimeout=5' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -o 'ServerAliveInterval=90' core@\$BOOTSTRAP_PROVISIONING_IP TAR_FILE=/tmp/log-bundle-bootstrap.tar.gz sudo -E /usr/local/bin/installer-gather.sh --id bootstrap &&
+  # Pass master IPs to installer-gather script to collect info from nodes which didn't join the cluster
+  NODE_IPS=()
+  for (( n=0; n<\$NUM_MASTERS; n++ ))
+  do
+     node_name=\$(printf \$MASTER_HOSTNAME_FORMAT \$n)
+     node_ip=\$(sudo virsh net-dumpxml \$BAREMETAL_NETWORK_NAME | xmllint --xpath "string(//dns[*]/host/hostname[. = '\${node_name}']/../@ip)" -)
```

Comment:
Seems like master/worker records at not defined in the dns section (at least I couldn't find it in the sos-report of the provisioning host). maybe something like this could work:
```suggestion
     node_ip=\$(sudo virsh net-dumpxml \$BAREMETAL_NETWORK_NAME | xmllint --xpath "string(//ip/dhcp[*]/host[@name='\${node_name}']/@ip)" -)
```